csharp/test/Drivers/Snowflake/DriverTests.cs

```diff
@@ -33,11 +33,30 @@ namespace Apache.Arrow.Adbc.Tests.Drivers.Interop.Snowflake
     /// queries to run.
     /// </remarks>
     [TestCaseOrderer("Apache.Arrow.Adbc.Tests.Xunit.TestOrderer", "Apache.Arrow.Adbc.Tests")]
-    public class DriverTests
+    public class DriverTests : IDisposable
```

Comment:
I'm not familiar with Xunit -- will it try to Dispose() of test classes for you if they are marked disposable?